GraalVM/CE/Dockerfile.ol8

```diff
@@ -0,0 +1,37 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2015 Oracle and/or its affiliates. All rights reserved.
+#
+
+FROM oraclelinux:8-slim
+
+# Note: If you are behind a web proxy, set the build variables for the build:
+#       E.g.:  docker build --build-arg "https_proxy=..." --build-arg "http_proxy=..." --build-arg "no_proxy=..." ...
+
+ARG GRAALVM_VERSION=20.0.0
+ARG JAVA_VERSION=java8
+ARG GRAALVM_PKG=https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-$GRAALVM_VERSION/graalvm-ce-$JAVA_VERSION-linux-amd64-$GRAALVM_VERSION.tar.gz 
+ENV LANG=en_US.UTF-8 JAVA_HOME=/opt/graalvm-ce-$JAVA_VERSION-$GRAALVM_VERSION/
+
+RUN microdnf update -y oraclelinux-release-el8 \
+    && microdnf --enablerepo ol8_appstream install -y bzip2-devel ed gcc gcc-c++ gcc-gfortran gzip file fontconfig less libcurl-devel make openssl openssl-devel readline-devel tar vi which xz-devel zlib-devel \
```

Comment:
The `ol8_appstream` repo is enbled by default, so you don't need to specify it here.